src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,47 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
-    @Override
+
+    private int size = 0;
```

Comment:
0 - default  value for int 
```suggestion
    private int size;
```